src/encoder/encoder.h

```diff
@@ -126,6 +128,6 @@ NAN_METHOD(encodeToGifBuffer);
 void pngWriteCB(png_structp png_ptr, png_bytep data, png_size_t length);
 int gifWriteCB(GifFileType * gif, const GifByteType * chunk, int len);
 void remapTransparentPixels(unsigned char * target, const unsigned char * map, size_t width, size_t height, int transColor, int threshold);
-void initAll(Handle<Object> exports);
+void initAll(v8::Local<v8::Object> exports);
```

Comment:
Not an owner for this project or anything, but I'd personally appreciate it if we removed the `using namespace <v8, node, etc>`; statements above and make API namespaces explicit everywhere in the module, as is done here.

👍